src/main/java/uk/ac/ebi/ddi/xml/validator/utils/Utils.java

```diff
@@ -49,26 +51,32 @@ public static List<Tuple> validateSemantic(Entry entry) {
                             "["+ ERROR+"]" + COLON + REPORT_SPACE + NOT_FOUND_MESSAGE + REPORT_SPACE +
                             DSField.DESCRIPTION));
         }
-
-        if (entry.getDates() != null && !entry.getDates().isEmpty()) {
-            Field.getFields().add(DSField.Date.PUBLICATION);
-            List<Field> fields = Field.getValuesByCategory(FieldCategory.DATE, FieldType.UNKNOWN);
-            for (Field field : fields) {
-                String errorCode = (field.getType() == FieldType.MANDATORY) ? ERROR : WARN;
+        if (entry.getDates() != null) {
+            List<Field> dateFieldList = new ArrayList<Field>();
```

Comment:
You could use `List.of(DSField.Date.PUBLICATION, DSField.Date.PUBLICATION_UPDATED... )` instead of adding them one at a time.